test/index.ts

```diff
@@ -10,7 +10,7 @@ describe('Example class', () => {
 	});
 	it('should return whatever is passed to exampleMethod()', () => {
 		const example: Example = new Example();
-		const param: string = 'This is my param.';
```

Comment:
Why was the type removed here?

Reply:
It is type inference the type is automatically set to string type

[Type Inference](https://www.typescriptlang.org/docs/handbook/type-inference.html)